solidity/test/integration/FullFlow.test.ts

```diff
@@ -73,6 +74,14 @@ describeFn("Integration Test - Full flow", async () => {
       governance,
       dkgResultChallengePeriodLength
     )
+
```

Comment:
Should we add a note that:
```
// Disable the reveal ahead period since refund locktimes are fixed
// within transactions used in this test suite.
```
?

Same comment to `integration/Slashing.test.ts`.

Reply:
See 997d2219